frontend/libs/features/calculators/src/components/calculator/calculator.component.ts

```diff
@@ -110,8 +112,21 @@ export class CalculatorComponent implements OnInit, OnChanges, OnDestroy {
 
   pdf(hidePdfComment: boolean) {
     this.hidePdfComment = hidePdfComment;
-    setTimeout(function () {
+
+    setTimeout(() => {
+      const styleElement = document.createElement('style');
+      document.head.appendChild(styleElement);
+
+      const printStyles = `
+      @media print {
```

Comment:
Kun je dit niet in `apps/smart-energy-web/src/styles.css` neerzetten? Nu voeg je elke keer een nieuwe css header toe wanneer je op print klikt, is wat minder netjes